src/rez/build_system.py

```diff
@@ -143,12 +167,12 @@ def __init__(self, working_dir, opts=None, package=None,
         self.opts = opts
 
     @classmethod
-    def is_valid_root(cls, path):
+    def is_valid_root(cls, path: str, package=None) -> bool:
         """Return True if this build system can build the source in path."""
```

Comment:
sub-classes are expected to have the `package` argument.  I think this was just an oversight on the base class.